File: SISTEMA VET/datos/conexion.vb
Imports System.Data.SqlClient
Imports System.Configuration
Public Class conexion
    Protected cnn As New SqlConnection

    Protected Function conectado()
        Try
            cnn = New SqlConnection("data source=DESKTOP-1AM46KO;initial catalog=Control_de_asistencia;integrated security=true")

            cnn.Open()
            Return True
        Catch ex As Exception
            MsgBox(ex.Message)
            Return False
        End Try
    End Function


    Protected Function desconectado()
        Try
            If cnn.State = ConnectionState.Open Then
                cnn.Close()
                Return True
            Else
                Return False
            End If
        Catch ex As Exception
            MsgBox(ex.Message)
            Return False
        End Try
    End Function
End Class

File: Datos/conexion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Data.SqlClient;
using System.Data;

namespace SIS_CA.Datos
{
    public class conexion
    {

        private SqlConnection Conexion = new SqlConnection("Data Source=DESKTOP-1AM46KO;Initial Catalog=Control_de_asistencia;Integrated Security=True");
        public SqlConnection AbrirConexion()
        {
            if (Conexion.State == ConnectionState.Closed)
                Conexion.Open();
            return Conexion;
        }
        public SqlConnection CerrarConexion()
        {
            if (Conexion.State == ConnectionState.Open)
                Conexion.Close();
            return Conexion;
        }
    }
}